src/OpenSpaceNet.cpp

```diff
@@ -39,6 +39,7 @@
 #include <imagery/SlidingWindowChipper.h>
 #include <imagery/DgcsClient.h>
 #include <imagery/EvwhsClient.h>
+#include <utility/DcMath.h>
```

Comment:
We do have `using dg::deepcore::almostEq;` but isn't used anywhere.  Can that be removed instead of this added?